src/ekat/util/ekat_feutils.hpp

```diff
@@ -0,0 +1,56 @@
+#ifndef EKAT_FEUTILS_HPP
+#define EKAT_FEUTILS_HPP
+
+#include "ekat/ekat.hpp"
+
+#ifdef EKAT_NEEDS_FEENABLEEXCEPT
+
+#include <fenv.h>
```

Comment:
I think this include can be moved outside the ifdef. If we move all the stuff that requires cfenv from ekat_assert.hpp to ekat_assert.cpp, then we can include fenv stuff just once. I can add a commit on this.

Reply:
Yeah, I just wanted to be certain that none of the new stuff conflicted with anything else that should have precedence.    Seems like it's ok.